libs/vre/pages/ontology/ontology-classes/src/lib/list-view/list-view.component.ts

```diff
@@ -93,12 +103,13 @@ export class ListViewComponent implements OnChanges, OnInit, OnDestroy {
     private _route: ActivatedRoute,
     private _router: Router,
     private _cd: ChangeDetectorRef,
-    private _store: Store
+    private _store: Store,
+    private _translateService: TranslateService
   ) {}
 
   ngOnInit(): void {
     this.componentCommsSubscriptions.push(
-      this._componentCommsService.on([Events.resourceLanguageChanged], () => this.doSearch()),
+      this.translation$.subscribe(() => this.doSearch()),
```

Comment:
unsubscribe onDestroy?
ngOnDestroy(): void {
  this.componentCommsSubscriptions.forEach(sub => sub.unsubscribe());
}


And we could remove this.ngUnsubscribe by replacing all takeUntil(this.ngUnsubscribe)) with take(1) I think.